content/_en/policy/contact.md

```diff
@@ -3,32 +3,53 @@ layout: contact_us
 sidenav: contact_us
 title: Contact us
 description: Have a question or a problem with Login.gov?
-scripts:
-  - /assets/js/build/contact.js
-intro_content: |-
-  Login.gov is here to help you create, sign in and manage your Login.gov account.
+intro_content: >-
+  Login.gov is here to help you create, sign in and manage your Login.gov
+  account.
+
 
   As a security measure, we do not have access to delete your Login.gov account or change your password on your behalf.
 
+
   Our operating hours are Monday - Friday, 8:00 a.m. - 8:00 p.m. ET. Please note that it could take up to two business days for our customer support agents to respond to your question. Thank you for your patience.
 
+
   If you need assistance with your Login.gov account, check out our help center articles for help with common issues.
 
+
   * [I can’t sign in to my account](/help/trouble-signing-in/overview/)
+
   * [I need help verifying my identity](/help/verify-your-identity/overview/)
+
   * [I need to change my information or manage my account](/help/manage-your-account/overview/)
+
   * [Browse more help articles](/help/)
+
   {: .help-question-list}
 
+
   ## Do you have a question about your agency account?
 
+
   Login.gov is a sign in service to access government agency websites. If you have questions about the agency website, which may include questions about your application status, membership, eligibility, benefits or other specific concerns related to your account with that government agency, please contact that agency.
-footer_content: |-
+footer_content: >-
   ## Partner with Login.gov
+
+
   Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) for more information.
 
+
   ## Report an issue
 
+
   If you want to report an issue, please review our [vulnerability disclosure policy](https://18f.gsa.gov/vulnerability-disclosure-policy/) and contact us using our [vulnerability disclosure form](https://docs.google.com/forms/d/e/1FAIpQLScuo4xCzBlpLnoq7-bDAVAxtJci03by7S-Q-Z_JUBDloK01QA/viewform).
```

Comment:
Is this text staying the same? I notice it includes most of the same information as the new "Report a Security or Privacy Issue" section.

Reply:
Yeah, the text is staying the same. I had noticed this in [this slack thread about the ticket](https://gsa-tts.slack.com/archives/C01710KMYUB/p1625762952041500) and noted that I would add the content. I am waiting for further review from @juliaelman if this is what she was thinking as Danielle is OOO this week.